cmake/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -102,6 +102,23 @@ else()
   hunter_fatal_error("TODO: set toolset for boost" WIKI "error.boost.toolset")
 endif()
 
+if("@ANDROID@")
+  set(toolset_version ndk)
```

Comment:
Agreed, it would make the association between toolset_name and toolset_version more clear.